prac_05/word_occurrences.py

```diff
@@ -47,4 +47,14 @@ def get_sorted_dict(word_to_occurrence: dict):
     return sorted_word_to_occurrence
 
 
+def get_max_word_length(word_to_occurrence: dict):
+    words = []
+    word_lengths = []
+    for word in word_to_occurrence:
+        words.append(word)
+        word_lengths.append(len(word))
+    max_word_length = max(word_lengths)
+    return max_word_length
+
+
 main()
```

Comment:
Need line after main.

Reply:
It does have one in pycharm, I think GitHub just removed it.